### ISSUE TITLE
Add the ability to have plugin aliases

### ISSUE THREAD

**Issue by michaelorr** [shaunduncan/helga#146]
There is the ability to alias a command inside a plugin, but there should be a way to alias the plugin name as well.
This would be useful for the poems plugin, for instance many people attempt to do:
`!plugins enable haiku` but this fails because there is no haiku plugin.
A small tweak to the way in which the operator plugin looks for matches in the plugins listing should make this possible.

**Comment by shaunduncan** [comment]
Yes, I had this same thought when dealing with plugin helps. This might be difficult though if there isn't a direct mapping from command name and plugin entry point name

**Comment by michaelorr** [comment]
I wasnt imagining the abilty to auto-map command names to entry point names... but more of a plugin setting
`plugin_aliases = ['haiku', 'tanka']`
manually defined in the plugins __init__.py...
When helga looks for a plugin matching `"some_string"` it first looks through the entry points it knows, failing that it looks through the `plugin_aliases` it knows. So authors could add aliases when they know there may be confusion in the users's mind.
This wouldnt solve the issue of collision at the alias level, but computering is hard™.

**Comment by bigjust** [comment]
I took a shot at such a thing https://github.com/bigjust/helga-alias
